lib/content-extractor.js

```diff
@@ -177,6 +177,7 @@ ContentExtractor.preprocess.OPERATIONS = {
         '.holder',
     ],
     mergeNodes: ['.article-text', '.section-inner', 'div.page', 'div.gtxt_body'],
+    propagateDirProperty: ['.direction'],
```

Comment:
This is the class for the div where the BBC article puts their direction information - it probably won't be the same for all websites. I could also do something like "Check all the divs!" but wanted to start with the most lightweight approach and see how it aligns with other sites.